Default.sublime-commands

```diff
@@ -30,7 +30,7 @@
         "caption": "Sync Settings: Edit User Settings",
         "command": "open_file",
         "args": {
-            "file": "${packages}/User/SyncSettings.sublime-settings"
+            "file": "${packages}/User/SyncSettings.sublime-settings",
```

Comment:
i am not sure if `sublime-commands` files support trailing comma, please remove it.